store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreCacheBackwardCompatibilityTest.java

```diff
@@ -1,88 +0,0 @@
-/*
- * Copyright 2020-2023 Crown Copyright
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.gchq.gaffer.federatedstore;
-
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-import uk.gov.gchq.gaffer.cache.CacheServiceLoader;
-import uk.gov.gchq.gaffer.cache.impl.JcsCacheService;
-import uk.gov.gchq.gaffer.cache.util.CacheProperties;
-import uk.gov.gchq.gaffer.graph.Graph;
-import uk.gov.gchq.gaffer.graph.GraphConfig;
-
-import java.util.HashSet;
-import java.util.Properties;
-import java.util.Set;
-
-import static java.util.Arrays.asList;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.ACCUMULO_STORE_SINGLE_USE_PROPERTIES;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.GRAPH_ID_MAP;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.SCHEMA_EDGE_BASIC_JSON;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.loadAccumuloStoreProperties;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.loadSchemaFromJson;
-import static uk.gov.gchq.gaffer.federatedstore.FederatedStoreTestUtil.resetForFederatedTests;
-import static uk.gov.gchq.gaffer.user.StoreUser.AUTH_1;
-import static uk.gov.gchq.gaffer.user.StoreUser.AUTH_2;
-import static uk.gov.gchq.gaffer.user.StoreUser.AUTH_USER_ID;
-
-public class FederatedStoreCacheBackwardCompatibilityTest {
-
-    private static final String ADDING_USER_ID = AUTH_USER_ID;
-    private static final String BACKWARDS_COMPATABILITY_2_0_0 = "backwards_compatability_2.0.0";
-    public static final String GAFFER_2_0_0_CACHE_CACHE_CCF = "src/test/resources/gaffer-2.0.0-cache/cache.ccf";
```

Comment:
Doesn't this need to be added back but with 2.1.0 now?

Reply:
I think @GCHQDev404 wanted to remove all of this based on the cache not needing to be backwards compatible because it is a cache and shouldn't be a long term store.

Reply:
I think the reality in Gaffer is that it **is** a long term store but is incorrectly called a cache...
How are users expected to keep track of their NamedOperations long term? We can't really expect them to manually note them down somewhere.

Reply:
To be done in https://github.com/gchq/Gaffer/issues/3031